views/templates/admin/main.tpl

```diff
@@ -22,168 +22,438 @@
-        <p>{l s='It only takes a few minutes to get started with Youstice.'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</p>
         <p>{l s='Already have a Youstice account?'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}
-            <input type="radio" name="have_account" id="haveAccountNo" value="0" checked="checked">
-            <label for="haveAccountNo">{l s='No'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</label>
-            <input type="radio" name="have_account" id="haveAccountYes" value="1">
-            <label for="haveAccountYes">{l s='Yes'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</label></p>
+            <label for="haveAccountNo">
+                <input type="radio" name="have_account" id="haveAccountNo" value="0" checked="checked">
+                {l s='No'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}
+            </label>
+            <label for="haveAccountYes">
+                <input type="radio" name="have_account" id="haveAccountYes" value="1">
+                {l s='Yes'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}
+            </label>
+        </p>
     </div>
 
-    <div class="yBlock stopScathingReviews">
+    <div class="yBlock yInfo">
+        {if $languageCode == 'fr'}
+            <h2>OPTEZ POUR UN SERVICE CLIENT D'EXCELLENCE</br> AVEC YOUSTICE, MANAGEZ LA SATISFACTION CLIENT</h2>
+            <p>&nbsp;</p>
+        {else}
+            <h2>{l s='SAY GOOD-BYE TO UNHAPPY CUSTOMERS WITH YOUSTICE.'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</h2>
+            <h3>{l s='SAY HELLO TO GREAT CUSTOMER CARE!'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</h3>
+        {/if}
+        <p>{l s='Thank you for downloading the Youstice module! It will help your business manage customer complaints and deal with unresolved disputes.'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</p>
+
+        <p>{l s='Youstice is designed to resolve customer disputes in a quick, emotion-free way, avoiding lengthy conversations or negative sentiment across social networks. Shoppers express their issue and propose a solution, choosing from options directly through the application. Your staff then responds and negotiates a win-win resolution, collecting valuable insight on how to improve the quality of your services.'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</p>
+
+        {if $languageCode == 'fr'}
+            <div class="col col-50 no-bottom-margin">
+                <img src="{$modulePath|escape:'htmlall':'UTF-8'}views/img/girl-fr.jpg"/>
+                <h2>{l s='RESOLVE CUSTOMER COMPLAINTS WITH YOUSTICE'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</h2>
+            </div>
+            <div class="col col-50 no-bottom-margin">
+                <img src="{$modulePath|escape:'htmlall':'UTF-8'}views/img/notebook-fr.jpg"/>
+                <h2>COUVREZ LES COÛTS DE MÉDIATION AVEC YOUSTICE</h2>
```

Comment:
Even if we can find ```{if $languageCode == 'fr'}```, you should add the translation tags around the displayed text.